docker-compose.yml

```diff
@@ -16,5 +16,11 @@ services:
       - ./:/app
     profiles: ["exclude-from-up"]
 
-  redis:
-    image: "redis/redis-stack-server:latest"
+  redisStack:
```

Comment:
```suggestion
  redis-stack:
```
I just noticed it should probably be redis-stack like the above auto-gpt to follow the usual docker-compose pattern